### DISCUSSION TITLE
clk dump

### DISCUSSION THREAD

**Discussion by firepower9966** [xjtuecho/EBAZ4205#31] [Development]
here is my clk dump from linux terminal. I have added missing Y3 25Mhz xtal and X5 50Mhz osc. Can some one post the default clk dump.

Usage:
clk dump - Print clock frequencies
zynq-uboot> clk dump
clk             frequency
    armpll          1333333320
    ddrpll          1066666656
     iopll           999999990
 cpu_6or4x           666666660
 cpu_3or2x           333333330
    cpu_2x           222222220
    cpu_1x           111111110
    ddr_2x           355555552
    ddr_3x           533333328
       dci            10158730
     lqspi            33333333
       smc            99999999
      pcap           199999998
      gem0            25000000
      gem1            16666667
     fclk0           124999999
     fclk1            25000000
     fclk2            41666666
     fclk3            41666666
     sdio0            50000000
     sdio1            50000000
     uart0            50000000
     uart1            50000000
      spi0           166666665
      spi1           166666665
 usb0_aper           111111110
 usb1_aper           111111110
 gem0_aper           111111110
 gem1_aper           111111110
sdio0_aper           111111110
sdio1_aper           111111110
 spi0_aper           111111110
 spi1_aper           111111110
 can0_aper           111111110
 can1_aper           111111110
 i2c0_aper           111111110
 i2c1_aper           111111110
uart0_aper           111111110
uart1_aper           111111110
 gpio_aper           111111110
lqspi_aper           111111110
  smc_aper           111111110
   dbg_trc            66666666
   dbg_apb            66666666
zynq-uboot>

**Comment by DavidJRichards** [comment]
An unmodified board gives this clk dump:
Model: Zynq Zed Development Board
Net:   Gem.e000b000
Hit 'd' to stop autoboot(eric): 0  
zynq-uboot> clk dump
clk             frequency
    armpll          1333333320
    ddrpll          1066666656
     iopll           999999990
 cpu_6or4x           666666660
 cpu_3or2x           333333330
    cpu_2x           222222220
    cpu_1x           111111110
    ddr_2x           355555552
    ddr_3x           533333328
       dci            10158730
     lqspi            33333333
       smc            99999999
      pcap           199999998
      gem0           999999990
      gem1            16666667
     fclk0           124999999
     fclk1            25000000
     fclk2            41666666
     fclk3            41666666
     sdio0            50000000
     sdio1            50000000
     uart0            50000000
     uart1            50000000
      spi0           166666665
      spi1           166666665
 usb0_aper           111111110
 usb1_aper           111111110
 gem0_aper           111111110
 gem1_aper           111111110
sdio0_aper           111111110
sdio1_aper           111111110
 spi0_aper           111111110
 spi1_aper           111111110
 can0_aper           111111110
 can1_aper           111111110
 i2c0_aper           111111110
 i2c1_aper           111111110
uart0_aper           111111110
uart1_aper           111111110
 gpio_aper           111111110
lqspi_aper           111111110
  smc_aper           111111110
   dbg_trc            66666666
   dbg_apb            66666666
zynq-uboot>

**Comment by firepower9966** [comment]
Seems the only difference is 
gem0 25000000 
gem0 999999990
